### PR TITLE
Add sites with details

### PR DESCRIPTION
**Pull request by joverlee521**
Adds sites with the types and categories described here: https://docs.google.com/document/d/1SqAlXuZttDXDdrRqdeSS910ecrM737MNRacBXHd6ePg/edit

The only site that is not on here is `Unknown` which I'm not entirely sure what to do with.
Also, `KaiserPermanente` is labeled as `clinic` but we have it as `retrospective`

### CODE REVIEW

**Comment by joverlee521** [comment]
[Revise site types/labels](https://trello.com/c/IzuwKzXh/265-revise-site-types-labels)

**Comment by kairstenfay** [comment]
> The only site that is not on here is `Unknown` which I'm not entirely sure what to do with.
> Also, `KaiserPermanente` is labeled as `clinic` but we have it as `retrospective`

This makes me wonder if there's both Kaiser Permanente retrospective data (from Mike Jackson) and a new Kaiser Permanente kiosk site (with data from REDCap). We should verify this before merging, perhaps with Robin in #informatics?

**Comment by joverlee521** [comment]
Putting this on hold until we get clarification about KaiserPermanente(see [slack](https://seattle-flu-study.slack.com/archives/CCAA9RBFS/p1579290122003900))

**Comment by kairstenfay** [comment]
> I find that Google doc pretty confusing and not very considered. For example, it labels `RetrospectiveHarborview` as a "workplace", which can't be right. Glad to see you've kept that "retrospective"!
> 
> Minor question: Why is `Harborview` labeled `longitudinalInpatient`?

I'll wager that the labels like 'Workplace' only apply to the 'Community' column. I do not believe the table is meant to be read across the Community/Clinic/Hospital columns. Rather, they are independent and only share the 'Adult'/'Child' labels.

**Comment by joverlee521** [comment]
> Minor question: Why is `Harborview` labeled `longitudinalInpatient`?

I based this off of Mike's [comment](https://seattle-flu-study.slack.com/archives/CL1Q38ATG/p1577470665018900?thread_ts=1577397176.011700&cid=CL1Q38ATG) and the fact that all samples associated with the site `Harborview` comes from `Inpatient Swabs and Blood Sample Records`

File: schema/deploy/warehouse/site/data.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        ('RetrospectiveChildrensHospitalSeattle',   '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveHarborview',                 '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveNorthwest',                  '{"category": "hospital",   "type": "retrospective"}'),
+        ('RetrospectiveUWMedicalCenter',            '{"category": "hospital",   "type": "retrospective"}'),
+        ('SeaTacDomestic',                          '{"category": "community",  "type": "publicSpace"}'),
+        ('SeaTacInternational',                     '{"category": "community",  "type": "publicSpace"}'),
+        ('SeattleCenter',                           '{"category": "community",  "type": "publicSpace"}'),
+        ('self-test',                               '{"category": "community",  "type": "self-test"}'),
+        ('StMartins',                               '{"category": "community",  "type": "shelter"}'),
+        ('swabNSend',                               '{"category": "community",  "type": "self-test"}'),
```
Comment (kairstenfay) [inline schema/deploy/warehouse/site/data.sql]: I think that the metrics team is interested in separating self-test and swab-n-send, per [their community break-down table](https://docs.google.com/spreadsheets/d/1eAmWEULmgogciQnQtYEaTGFnl9hkt7snTgF0u-xNTi4/edit#gid=10263779).

Comment (joverlee521) [reply schema/deploy/warehouse/site/data.sql]: I feel like that should be looking at the site identifier itself? They are both "self-test" type

Comment (tsibley) [reply schema/deploy/warehouse/site/data.sql]: Maybe `self-swab` is a more accurate/precise type? Both study arms swab themselves, but only the `self-test` / home test arm is actually _testing_ themselves with an at-home device (though that's on hold right now!).

But maybe introducing another term is more confusing...

Comment (kairstenfay) [reply schema/deploy/warehouse/site/data.sql]: > I feel like that should be looking at the site identifier itself? They are both "self-test" type

I'm just thinking keeping self-test and swab-n-send separate will make metrics calculations much easier, seeing as the current site types are all defined by metrics-related needs.

Comment (joverlee521) [reply schema/deploy/warehouse/site/data.sql]: Taking another look at https://docs.google.com/document/d/1SqAlXuZttDXDdrRqdeSS910ecrM737MNRacBXHd6ePg/edit

They have `Swab-and-send` and `Swab-and-send with self-test` labeled as `Home` for site_type

Comment (tsibley) [reply schema/deploy/warehouse/site/data.sql]: > seeing as the current site types are all defined by metrics-related needs

Nod, though not just metrics, but also modeling and the Nextstrain build.

Comment (tsibley) [reply schema/deploy/warehouse/site/data.sql]: A site type of `home` makes sense to me.

**Pull request merged**